packages/@apphosting/common/src/index.ts

```diff
@@ -1,5 +1,58 @@
 import { spawn } from "child_process";
 
+// Output bundle specifications to be written to bundle.yaml
+export interface OutputBundle {
+  version: "v1alpha";
+  serverConfig: ServerConfig;
+  metadata: Metadata;
+}
+
+// Fields needed to configure the App Hosting server
+interface ServerConfig {
+  // Command to start the server (e.g. ["node", "dist/index.js"]) assume this command is run from the root dir of the workspace
+  runCommand: string[];
+  // Environment variable present in the server execution environment.
```

Comment:
Is this implying Cloud Run (eg runtime)? Do we need to change this comment if we ever want to support buildtime?